static/src/javascripts-legacy/projects/common/modules/atoms/youtube.js

```diff
@@ -71,7 +75,41 @@ define([
         }
     }
 
+
+
+    function shouldAutoplay(){
+
+        function isMobile() {
+            return detect.isIOS() || detect.isAndroid();
```

Comment:
is this really the check you want? isMobile doesn't really seem to equal being iOS or android does it? do you want a breakpoint maybe?